examples/light-switch-app/nrfconnect/main/BindingHandler.cpp

```diff
@@ -0,0 +1,292 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "BindingHandler.h"
+#ifdef CONFIG_CHIP_LIB_SHELL
+#include "ShellCommands.h"
+#endif
+
+#include <logging/log.h>
+LOG_MODULE_DECLARE(app);
+
+using namespace chip;
+using namespace chip::app;
+
+void BindingHandler::Init()
+{
+#ifdef CONFIG_CHIP_LIB_SHELL
+    SwitchCommands::RegisterSwitchCommands();
+#endif
+    chip::DeviceLayer::PlatformMgr().ScheduleWork(InitInternal);
```

Comment:
Minor: we don't need `chip::` here, as you have using namespace chip at the top of the file.